app/views/contact_methods/_whatsapp.html.erb

```diff
@@ -0,0 +1,32 @@
+<div class="contact-panel contact-panel__chat-open" id="js-whatsapp">
+  <h2 aria-level="2" class="contact-panel__heading" role="heading">
+    <%= render 'contact_methods/whatsapp_logo' %>WhatsApp
+  </h2>
+  <p class="contact-panel__chat-javascript">
+    <%= t('contact_panels.webchat.nojs_html') %></a>.
```

Comment:
This text appears to be missing: get a `translation_missing` message in the markup.

Reply:
Fixed